watson_developer_cloud/conversation_v1.py

```diff
@@ -11,108 +13,196 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 """
-The IBM Watson Conversation service combines machine learning, natural
-language understanding, and integrated dialog tools to create conversation
-flows between your apps and your users.
+The IBM Watson Conversation service combines machine learning, natural language
+understanding, and integrated dialog tools to create conversation flows between your apps
+and your users.
 """
 
+import json
+from .watson_developer_cloud_service import WatsonInvalidArgument
 from .watson_developer_cloud_service import WatsonDeveloperCloudService
 
+##############################################################################
+# Service
+##############################################################################
+
+
 class ConversationV1(WatsonDeveloperCloudService):
     """Client for the Conversation service."""
```

Comment:
Add API version to the description.
```
Client for the Conversation V1 service.
```

Reply:
or maybe 
```
Coversation V1 service.
```

Reply:
I like "The Conversation V1 service".  I will fix.